dapp/src/pages/DashboardPage/AcreTVLProgress.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+import {
+  StackProps,
+  useMultiStyleConfig,
+  HStack,
+  VStack,
+  Grid,
+  Box,
+} from "@chakra-ui/react"
+import { useTVL } from "#/hooks"
+import { BitcoinIcon } from "#/assets/icons"
+import { CurrencyBalance } from "#/components/shared/CurrencyBalance"
+import ProgressBar from "#/components/shared/ProgressBar"
+import { TextMd, TextXs } from "#/components/shared/Typography"
+
+type AcreTVLProgressProps = StackProps
+
+export function AcreTVLProgress(props: AcreTVLProgressProps) {
+  const styles = useMultiStyleConfig("AcreTVLProgress")
+  const totalValueLocked = useTVL()
+
+  return (
+    <Box sx={styles.container} {...props}>
+      <HStack sx={styles.wrapper}>
+        <Grid sx={styles.contentWrapper}>
+          <BitcoinIcon sx={styles.valueIcon} />
+
+          <CurrencyBalance
+            size="3xl"
+            amount={totalValueLocked.value}
```

Comment:
Currently, we show `244.6570`. Can we reduce the number of decimals? How about we show two decimals?

Reply:
Ref commit: 880130a9